src/themes/greyVest.js

```diff
@@ -692,7 +692,11 @@ export let GVStyle = () => (
     `}
   </style>
 )
-export let Table = x => <table className="gv-table" {...x} />
+export let Table = x => (
+  <div style={{ overflow: 'auto' }}>
```

Comment:
Based on your PR description should this be `scroll auto`?

Reply:
Not the best use of a pseudo-regex, I meant something like `overflow: auto` or `overflow: scroll`, but `auto` is nicer since it only shows the scrollbars when necessary.